br/br-pitr-guide.md

```diff
@@ -0,0 +1,133 @@
+--storage 's3://backup-101/snapshot-${date}?access_key=${access_key}&secret_access_key=${secret_access_key}"'
+```
+
+## Run PITR
+
+To restore the cluster to any point in time within the backup retention period, you can use `br restore point`. When you run this command, you need to specify the **time point you want to restore**, **the latest snapshot backup data before the time point**, and the **log backup data**. `br` will automatically determine and read data needed for the restore, and then restore these data to the specified cluster in order.
+
+```shell
+br restore point --pd "${PD_IP}:2379" \
+--storage='s3://backup-101/logbackup?access_key=${access_key}&secret_access_key=${secret_access_key}"' \
+--full-backup-storage='s3://backup-101/snapshot-${date}?access_key=${access_key}&secret_access_key=${secret_access_key}"' \
+--restored-ts '2022-05-15 18:00:00+0800'
+```
+
+During data restore, you can view the progress through the progress bar in the terminal. The restore is divided into two phases, full restore and log restore (restore meta files and restore KV files). After each phase is completed, `br` outputs information such as restore time and data size.
+
+```shell
+Full Restore <--------------------------------------------------------------------------------------------------------------------------------------------------------> 100.00%
+*** ["Full Restore success summary"] ****** [total-take=xxx.xxxs] [restore-data-size(after-compressed)=xxx.xxx] [Size=xxxx] [BackupTS={TS}] [total-kv=xxx] [total-kv-size=xxx] [average-speed=xxx]
+Restore Meta Files <--------------------------------------------------------------------------------------------------------------------------------------------------> 100.00%
+Restore KV Files <----------------------------------------------------------------------------------------------------------------------------------------------------> 100.00%
+*** ["restore log success summary"] [total-take=xxx.xx] [restore-from={TS}] [restore-to={TS}] [total-kv-count=xxx] [total-size=xxx]
+```
+
+## Clean up outdated data
+
+As described in the [Usage Overview of TiDB Backup and Restore](/br/br-use-overview.md):
+
+To perform PITR, you need to restore the full backup before the restore point, and the log backup between the full backup point and the restore point. Therefore, for log backups that exceed the backup retention period, you need to run `br log truncate` to delete the backup before the specified time point. **It is recommended to only delete the log backup before the full snapshot**.
+
+The following steps describe how to clean up backup data that exceeds the backup retention period:
+
+1. Get the **last full backup** outside the backup retention period.
+2. Use the `validate` command to get the time point corresponding to the backup. Assume that the backup data before 2022/09/01 needs to be cleaned, you should look for the last full backup before this and ensure that it will not be cleaned.
```

Comment:
```suggestion
2. Use the `validate` command to get the time point corresponding to the backup. Assume that the backup data before 2022/09/01 needs to be cleaned, you should look for the last full backup before this time point and ensure that it will not be cleaned.
```